.travis.yml

```diff
@@ -406,6 +406,12 @@ script:
   - opt_enable_adv AUTOMATIC_CURRENT_CONTROL STEALTHCHOP
   - build_marlin
   #
+  # M43 pins_debugging
+  #
+  - restore_configs
+  - opt_enable PINS_DEBUGGING
+  - build_marlin
```

Comment:
If this is moved into one of the other builds, then it will make Travis CI finish more quickly — one less build.